app/controllers/flashcards_controller.rb

```diff
@@ -1,5 +1,11 @@
 class FlashcardsController < ApplicationController
   def index
-    @card = Card.dated.sort_random.last
+    @card = if current_user && current_user.decks.find_current_deck.nil?
```

Comment:
In case we don't have deck for user. we should respond like a `please set current deck`